backend/core/src/listings/tests/listings.service.spec.ts

```diff
@@ -392,4 +392,16 @@ describe("ListingsService", () => {
       )
     })
   })
+
+  describe("getListingsExternal", () => {
```

Comment:
nit -- this describe block should be the`listings.service.ts` function name, `listIncludeExternal`